packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/NftForm.utils.ts

```diff
@@ -21,6 +21,10 @@ export const createValidationSchema = (data: NftFormFields) => {
     .nullable()
     .optional()
 
+  const buyNowPrice = z
+    .number({ required_error: 'Buy now price must be provided', invalid_type_error: 'Buy now price must be a number' })
+    .min(1, 'Buy now price cannot be lower than 1')
```

Comment:
We still need to cover one case. Minimum bid shouldn't be higher than buy now price: 
![image](https://user-images.githubusercontent.com/51168865/159489582-570ff424-f0bf-4a14-b48a-ec754f477686.png)